apps/backend/src/reconciliation/cash-reconciliation.service.ts

```diff
@@ -157,11 +137,11 @@ export class CashReconciliationService {
         location
       );
     this.appLogger.log(
-      `${aggregatedPayments.length} AGGREGATED PAYMENTS PENDING RECONCILIATION`,
+      `${aggregatedPayments?.length} AGGREGATED PAYMENTS PENDING RECONCILIATION`,
```

Comment:
If this nullish is necessary, then I imagine you'd want `aggregatedPayments?.length || 0`. Otherwise, the log won't look right!

Reply:
Oops, good call! On this topic, I was wondering if I should scale back the logs? Or maybe add more logging to the specific services that I are querying for/updating the data vs having the logs here? The logs are bit unwieldily...

Reply:
Logs are cheap, no need to scale back, but we may want to refine them so as to make them easier to sift through.